config/kubernetes/sig-openstack/teams.yaml

```diff
@@ -2,36 +2,42 @@ teams:
   provider-openstack-api-reviews:
     description: ""
     maintainers:
-    - idvoretskyi
+    - lingxiankong
```

Comment:
Needs to be in `members`. Only org admins need to be listed as maintainers.

Reply:
So, should I just keep idvoretskyi in the maintainers for all items and adding myself to all the member groups?